regparser/grammar/unified.py

```diff
@@ -191,3 +191,14 @@ def appendix_section(match):
            + atomic.section
            + Optional(depth1_p)).setParseAction(keep_pos).setResultsName(
                "tail", listAllMatches=True)))
+
+notice_cfr_p = (
+    Suppress(atomic.title)
+    + Suppress("CFR")
+    + Suppress(atomic.part_marker | atomic.parts_marker)
```

Comment:
Would it make sense for this to be optional? I recall `12 CFR 1026` being used often